bigquery/cloud-client/user_credentials.py

```diff
@@ -47,10 +51,13 @@ def main(project, launch_browser=True):
     # [START bigquery_auth_user_query]
     from google.cloud import bigquery
 
-    # TODO: This project will be billed for the query processing. The user must
-    #       have the bigquery.jobs.create permission on this project to run a
-    #       query. See:
-    #       https://cloud.google.com/bigquery/docs/access-control#permissions
+    # TODO: Uncomment the `project` variable below.
```

Comment:
I think this should always have something like "and set it to your project ID"

Reply:
For the user auth example, it's actually the user's project ID, not the developer's.